hexrd/ui/image_file_manager.py

```diff
@@ -158,9 +158,10 @@ def hdf_path_exists(self, f):
     def hdf5_path_exists(self, f):
         all_paths = []
         if HexrdConfig().hdf5_path is not None:
-            all_paths.append(HexrdConfig().hdf5_path)
+            # added by SS 05/01/23 to check against empty list
+            if HexrdConfig().hdf5_path:
+                all_paths.append(HexrdConfig().hdf5_path)
```

Comment:
This can be simplified a bit further by dropping the `if HexrdConfig().hdf5_path is not None` condition to just be:
```
if HexrdConfig().hdf5_path:
    all_paths.append(HexrdConfig().hdf5_path)
```